jetstream_pt/cache_manager.py

```diff
@@ -106,13 +106,13 @@ def empty(cls, shape, device):
         return cls(k, v, 0, device)
 
 def KVCacheGenerate_flatten(cache):
-    return torch_xla2.tensor.unwrap((cache.cache_k, cache.cache_v)), (self.position, self.sharding)
+    return torch_xla2.tensor.unwrap((cache.cache_k, cache.cache_v)), (cache.pos, cache.sharding)
```

Comment:
This is the time we ask for unit tests. :) Thanks!

Reply:
Agree. This function is not used in any place, I will consider to remove this and other unused functions.